lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -40,6 +40,8 @@ public static function buttonFunction(array $params, $template)
         $href = isset($params['href']) ? trans($params['href']) : null;
         // optional - allow to easily attach event handler in jquery,
         $id = isset($params['id']) ? $params['id'] : null;
+        // optional - allow to open content in a new window or non-default target,
+        $target = isset($params['target']) ? $params['target'] : null;
```

Comment:
Sensowne!

Reply:
Da się to dalej zoptymalizować do jednej próby np. parametry target, externalid, accesskey, form, clipboard
Np `{button parm_target=wartosc1` w szablonie HTML przełożyłby się na kod `<button target=wartosc1`